### PR TITLE
Add new permission to grant 'add' on cas container

### PR DESCRIPTION
**Pull request by Tiboris**
https://pagure.io/freeipa/issue/6609

### CODE REVIEW

File: ipaserver/install/plugins/update_managed_permissions.py [hunk 1]
```diff
@@ -291,6 +291,14 @@
             'ipadomainlevel', 'objectclass',
         },
     },
+    'System: Grant add on CAS container': {
```
Comment (MartinBasti) [inline ipaserver/install/plugins/update_managed_permissions.py]: IMO this should be kept together with other sub CA related permissions in `ipaserver/plugins/ca.py`

File: ipaserver/plugins/ca.py [hunk 2]
```diff
@@ -138,6 +138,19 @@ class ca(LDAPObject):
             ],
             'default_privileges': {'CA Administrator'},
         },
+        'System: Grant add on CAS container': {
+            'ipapermtargetfilter': {'(objectclass=nscontainer)'},
+            'ipapermbindruletype': 'permission',
+            'ipapermright': {'add'},
+            'replaces':
```
Comment (MartinBasti) [inline ipaserver/plugins/ca.py]: You can remove `replaces`, there is no old ACI that should be removed

File: ipaserver/plugins/ca.py [hunk 3]
```diff
@@ -138,6 +138,19 @@ class ca(LDAPObject):
             ],
             'default_privileges': {'CA Administrator'},
         },
+        'System: Grant add on CAS container': {
```
Comment (MartinBasti) [inline ipaserver/plugins/ca.py]: I prefer to keep consistency with other privileges, please call it 'System: Grant add on CAs container'

File: ipaserver/plugins/ca.py [hunk 4]
```diff
@@ -138,6 +138,12 @@ class ca(LDAPObject):
             ],
             'default_privileges': {'CA Administrator'},
         },
+        'System: Grant add on CAs container': {
+            'ipapermtargetfilter': {'(objectclass=nscontainer)'},
+            'ipapermbindruletype': 'permission',
+            'ipapermright': {'add'},
+            'default_privileges': {'CA Administrator'},
+        },
```
Comment (frasertweedale) [inline ipaserver/plugins/ca.py]: I think we do not want to add a new permission for this; the existing 'System: Add CA' permission
should be sufficient.

**Comment by frasertweedale** [comment]
This problem is actually caused by a bug in a preliminary permission check during
`ca_add.pre_callback`, not an ACI issue.  I am looking into it, but this is not the
solution, so I am NACKing it.

**Comment by frasertweedale** [comment]
New PR: https://github.com/freeipa/freeipa/pull/837

**Pull request closed without merging**